### PR TITLE
Revealing the secret of DL models - Ch 7

### PR DESCRIPTION
**Pull request by tpalczew**
Added example notebook with eli5, shap, and lime 
Updated readme

### CODE REVIEW

File: Chapter_7/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    num_iterations = conf["num_it"]
+    for i in range(num_iterations):
+        … // training logic
+        tune.report(mean_accuracy=acc)
+
+tune.run(
+    run_or_experiment = tr_function
+    conf={ “num_it”: tune.grid_search([10, 20, 30, 40])})
+```
+want to find more? don't forget to buy our book.
```
Comment (ljj7975) [inline Chapter_7/README.md]: what? lool I wouldn't do this.

**Comment by ljj7975** [comment]
For Secret_of_DL_models.ipynb,

`data = pd.read_csv("/Users/tpalczew/Production-Ready-Applied-Deep-Learning/Chapter_3/sample_google_scholar.csv")`
You are okay keeping the tpalczew?


`shap.force_plot(explainer.expected_value, shap_values[0], data)`
seems to be failing

`IFrame(src='lime.html', width=700, height=600)`
is this supposed to be showing something?

Can you actually add a brief explanation for each of the results/visualization?

File: Chapter_7/README.md (hunk 1 above)
Comment (tpalczew) [reply Chapter_7/README.md]: why not???? LOL

Comment (tpalczew) [reply Chapter_7/README.md]: removed
